### PR TITLE
Add DF-SHOW

### PR DESCRIPTION
**Pull request by roberthawdon**
## Checklist for submitting a pull request to `Terminals Are Sexy`:

- [x] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/k4m4/terminals-are-sexy/blob/master/contributing.md) of this repo.
- [x] I have **searched** the [PRs](https://github.com/k4m4/terminals-are-sexy/pulls) of this repo and believe that this is not a duplicate.

 

- **Title**: DF-SHOW
- **Link**: https://github.com/roberthawdon/dfshow
- **Category**: Tools and Plugins
- **Description**: An interactive directory and file browser with a similar feel to the `ls` command.

### CODE REVIEW

**Comment by roberthawdon** [comment]
Additional endorsements:

[Linux Today](https://www.linuxtoday.com/):
[https://www.linuxtoday.com/developer/df-show-a-terminal-file-manager-based-on-an-old-dos-application-181029061007.html](https://www.linuxtoday.com/developer/df-show-a-terminal-file-manager-based-on-an-old-dos-application-181029061007.html)

[OS Technix](https://www.ostechnix.com/):
[https://www.ostechnix.com/df-show-a-terminal-file-manager-based-on-an-old-dos-application/](https://www.ostechnix.com/df-show-a-terminal-file-manager-based-on-an-old-dos-application/)

[Przystajnik](https://404.g-net.pl/) (Polish):
[https://404.g-net.pl/2018/10/terminal-pogryzl-czlowieka-df-show/](https://404.g-net.pl/2018/10/terminal-pogryzl-czlowieka-df-show/)

**Comment by roberthawdon** [comment]
It has also been further endorsed on page 80 of the March issue of Linux Magazine (issue 220):

http://www.linux-magazine.com/Issues/2019/220/FOSSPicks

File: readme.md [hunk 1]
```diff
@@ -212,6 +212,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [cointop](https://github.com/miguelmota/cointop) - The fastest and most interactive terminal based UI application for tracking cryptocurrencies.
 * [colorls](https://github.com/athityakumar/colorls) - Beautify the terminal's `ls` command, with color and font-awesome icons.
 * [ddgr](https://github.com/jarun/ddgr) - DuckDuckGo from the terminal.
+* [df-show](https://github.com/roberthawdon/dfshow) - An interactive directory and file browser with a similar feel to the `ls` command.
```
Comment (k4m4) [inline readme.md]: Please drop `An` --> instead, description should be `Interactive directory and file browser with a similar feel to the ls command.`

Comment (roberthawdon) [reply readme.md]: Thanks, I've updated that.

**Comment by k4m4** [comment]
Thanks! Could you also squash your commits into one? Apologies for the delayed response.

**Comment by roberthawdon** [comment]
Hi, apologies for the delay. That has been squashed as requested.

**Pull request merged**